zetaclient/tss/tss_keysign_manager.go

```diff
@@ -0,0 +1,49 @@
+package tss
+
+import (
+	"sync"
+
+	"github.com/rs/zerolog"
+	"github.com/zeta-chain/zetacore/zetaclient/metrics"
+)
+
+// KeySignManager is a manager to keep track of concurrent keysigns performed by go-tss
+type KeySignManager struct {
```

Comment:
We should avoid using `Manager` name for types as often too generic, unclear.

If it is solely used to track concurrent key signs, we may call it `ConcurrentKeysignsTracker`?

Also a small note:
using `KeySign` instead of `Keysign` is not consistent with `keysign` filename. If these are two separate word, the filename should contains underscore, but I would tend to keep it as a single word